### PR TITLE
Travis setup

### PR DESCRIPTION
**Pull request by pmedapuram**

### CODE REVIEW

**Comment by pmedapuram** [comment]
@adarsh-ramakrishna-sfdc @vamshi-sfdc 
take a look at this PR. I worked on this when I had some free time. This is a simple travis build that will run every time a person create a PR. As of now I am running our mvn test with all the secrets encrypted. 

Also added the build status link to the README.

**Comment by vamshi-sfdc** [comment]
+ 1

Please add all of us as admins to the travis account

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+env:
+  global:
+  - secure: ELavjRwFdNq/u+r/3q8CSI8YwpR3qAh2utkiUE43HjxIcxFhb16Y8yyZtqbdrHNrXHuXtTg6NPFo751I/08cfwlHaNLlhCiUc1mIOZGir0Y3qyJb2Idyu2SWDFU6Qok4qLuZLzCuizg9YOT2D0f7C3GcmYiSk2STg0LoZXt+4rxTHMX+qaQnzdN58o6dfstkMWFF402bh49QJmv6z0kfR6Jyxq40H2mEHrCVvA4UWE4kHbEk2GULui9mg4SKi9yx5yBrRhuAJcJsZq8aIwuApRgLuZdh5iBWUy1M9TLaFL7izC/mV+7j40k4fcOxaWjsP6D+2OnqMmi2p1v8M1l4cT6YuJGRx9MVDtNvC8jkQObWJF2+OuaFFtwZOIUlzscMNCh0PwrlsQgZQgzssYE8yOkiuRKNQq9+3W42+ZH+7iGzvu1O23JcSjhff7FKIWkmDlZhR55o0vtQS2M4uBVRWpgurMBPHcTgi72pzxEkKUNA1Bu0zAQSWMSwd7/3Z04fyCWbanJFm+gaPvPqBZlGtnwW1SyOBrgknvDDvlcgqAoKRC8M4VeYLxYlbsHIrLVPInWZ6KOv+6TPISoqOM/3AQPxUNcAuUTZnfRKNlKCB5kQxu3g4tcwARc47ZNhUSVX+XBtFVdDqlx2PrbDMQIWAMb1fAS8010jOcL2z3uE3j0=
```
Comment (vamshi-sfdc) [inline .travis.yml]: How do other open source projects deal with securing this content?

Comment (pmedapuram) [inline .travis.yml]: The recommended way by travis to encrypt secrets is by using 'secure' global env variables as given [here](https://docs.travis-ci.com/user/encryption-keys/). So followed the same method.

**Comment by pmedapuram** [comment]
@vamshi-sfdc  Everyone can login to travis with their github credentials, and if there are the admins of the repo, they can admin access to the travis build as well. 

github permissions ---> reflect on --> travis build for the corresponding repo.

**Comment by vamshi-sfdc** [comment]
in what scenarios can https://github.com/forcedotcom/ApexUnit/tree/travis status be unknown?

**Comment by pmedapuram** [comment]
The build status is unknown when there are no builds.

In this case, there are no builds on the master yet.

**Comment by vamshi-sfdc** [comment]
this is not master

**Comment by pmedapuram** [comment]
The link that the build status is pointing to is: _https://travis-ci.org/forcedotcom/ApexUnit.svg?branch=master_  which is pointing to the 'master' branch.

**Comment by pmedapuram** [comment]
@adarsh-ramakrishna-sfdc  @vamshi-sfdc  Can we merge this?

**Comment by adarsh-ramakrishna-sfdc** [comment]
The build passes now.. Looks good to merge.. :+1:  from me.

**Comment by pmedapuram** [comment]
@vamshi-sfdc

File: src/test/java/com/sforce/cd/apexUnit/arguments/CommandLineArgumentsTest.java [hunk 2]
```diff
@@ -113,7 +113,7 @@ public void getClassManifestFileLoc() {
 	}
 
 	public void getPassword() {
-		logs.info(" cmdLineArgs.getPassword() is " + CommandLineArguments.getPassword());
+		//logs.info(" cmdLineArgs.getPassword() is " + CommandLineArguments.getPassword());
```
Comment (vamshi-sfdc) [inline src/test/java/com/sforce/cd/apexUnit/arguments/CommandLineArgumentsTest.java]: Good to merge upon removing both the commented lines. Security risk @pmedapuram

**Comment by pmedapuram** [comment]
@adarsh-ramakrishna-sfdc  @vamshi-sfdc  Reviewed comments from you both. Good to merge.

**Pull request merged**